src/components/common/YourCompany/YourCompany.vue

```diff
@@ -237,12 +237,8 @@
 
       <div class="section-container">
         <v-row no-gutters>
-          <v-col cols="3">
-            <label><strong>Nature of Business</strong></label>
-          </v-col>
-
-          <v-col cols="9">
-            <div class="info-text">Cookie and cracker manufacturing</div>
+          <v-col>
```

Comment:
Do you know if this automatically acts like cols="12" ?

I get that you're doing this here to keep the formatting similar across all sections.